client/src/features/action/assets/Assets.tsx

```diff
@@ -131,8 +153,11 @@ function Assets(): JSX.Element {
               </Button>
             </NavLink>
           </Paper>
-          <NeedsAndOffers headerText="Nonprofit Needs" assets={mockData} />
-          <NeedsAndOffers headerText="Offers" assets={mockData} />
+          <NeedsAndOffers headerText="Nonprofit Needs" assets={mockData} />\
```

Comment:
Thanks! Removed the slash and extra findAll() method in assets.service

Reply:
Yes please! I'm assuming github's instructions won't work since the main branch is protected?
![image](https://user-images.githubusercontent.com/87346902/163282496-8bfd9897-a835-4173-947d-f7a401b34221.png)